### PR TITLE
feat: CNS writes SWIFT conflist

### PR DESCRIPTION
**Pull request by nddq**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

File: cns/cniconflist/generator_linux.go [hunk 1]
```diff
@@ -133,3 +133,31 @@ func (v *CiliumGenerator) Generate() error {
 
 	return nil
 }
+
+// Generate writes the CNI conflist to the Generator's output stream
+func (v *SWIFTGenerator) Generate() error {
+	conflist := cniConflist{
+		CNIVersion: overlaycniVersion,
```
Comment (rbtr) [inline cns/cniconflist/generator_linux.go:140]: not overlay/swift specific so we should change this variable to `azurecniVersion`

**Review by rbtr — APPROVED** [review]

File: cns/cniconflist/generator.go [hunk 2]
```diff
@@ -63,6 +66,11 @@ type CiliumGenerator struct {
 	Writer io.WriteCloser
 }
 
+// SWIFTGenerator generates the Azure CNI conflist for the SWIFT scenario
+type SWIFTGenerator struct {
```
Comment (vipul-21) [inline cns/cniconflist/generator.go:70]: Just curious, how/when we use this generation of conflist ?

Comment (rbtr) [reply cns/cniconflist/generator.go:70]: it will be used in aks swift

**Review by neaggarwMS — COMMENTED** [review]
Reviewing

File: cns/service/main.go [hunk 3]
```diff
@@ -93,6 +93,7 @@ const (
 	scenarioDualStackOverlay cniConflistScenario = "dualStackOverlay"
 	scenarioOverlay          cniConflistScenario = "overlay"
 	scenarioCilium           cniConflistScenario = "cilium"
+	scenarioSWIFT            cniConflistScenario = "swift"
```
Comment (thatmattlong) [inline cns/service/main.go:96]: Need to add this scenario to the switch on L542

**Review by thatmattlong — CHANGES_REQUESTED** [review]

**Review by rbtr — APPROVED** [review]

**Review by thatmattlong — APPROVED** [review]

**Pull request merged**